tests/python/pants_test/engine/test_scheduler.py

```diff
@@ -224,15 +240,55 @@ def test_get_type_match_failure(self):
 
   def test_unhashable_failure(self):
     """Test that unhashable Get(...) params result in a structured error."""
-    expected_msg = """TypeError: unhashable type: 'list'"""
-    with assert_execution_error(self, expected_msg):
+    with assert_execution_error(self, "TypeError: unhashable type: 'list'"):
```

Comment:
nit. Can we delete this test, or split it into two to separate the two failure conditions?

Reply:
Split the test method into two test methods, you mean? I wanted to have that assertion here to check that some appropriate error message is produced for non-datatype objects as well. It could be split into a separate method if necessary, although I see it as being closely related to the datatype `hash()` assertion, unless I'm missing something?

Reply:
I've deleted the test!